sessionctx/variable/tidb_vars.go

```diff
@@ -351,6 +351,9 @@ const (
 	// TiDBEnableChangeColumnType is used to control whether to enable the change column type.
 	TiDBEnableChangeColumnType = "tidb_enable_change_column_type"
 
+	// TiDBEnableMultiSchemaChange is used to control whether to enable the multi schema change.
+	TiDBEnableMultiSchemaChange = "tidb_enable_multi_schema_change"
```

Comment:
Maybe we should use `"tidb_enable_change_multi_schema"` like `"tidb_enable_change_column_type"`?

Reply:
good